lib/events/filesessions/fileasync.go

```diff
@@ -296,6 +296,10 @@ func (u *Uploader) Scan(ctx context.Context) (*ScanStats, error) {
 				u.log.DebugContext(ctx, "Recording was uploaded by another process.", "recording", fi.Name())
 				continue
 			}
+			if errors.Is(err, errNoEncryptedUploader) {
+				u.log.ErrorContext(ctx, "Skipped encrypted session recording due to missing uploader", "recording", fi.Name(), "error", err)
```

Comment:
Will including the error in the output add any meaningful/additional information to this log? The errNoEncryptedUploader is defined as `no encrypted uploader configured while uploading encrypted recording` which seems to be the same as the log message in different words.
```suggestion
				u.log.ErrorContext(ctx, "Skipped encrypted session recording due to missing uploader", "recording", fi.Name())
```